docs/whitelabelling.md

```diff
@@ -88,6 +88,7 @@ Things the AppTeam has to do and therefore probably create issues for.
 
 - Add build config
 - Setup app store presence
+- Add new App to permissions of ci service account  
```

Comment:
Is this google play specific? Or applies for apple as well?

Reply:
only for google play.

Reply:
```suggestion
- Add new App to permissions of GooglePlay CI service account  
```